com.jetbrains.IntelliJ-IDEA-Community.json

```diff
@@ -3,7 +3,7 @@
   "runtime": "org.freedesktop.Sdk",
   "runtime-version": "24.08",
   "sdk": "org.freedesktop.Sdk",
-  "command": "idea",
+  "command": "idea-wrapper",
```

Comment:
Can you leave this one as the original command from the IDE?

Notice this is how it's going to be displayed in the running processes.

Reply:
> Notice this is how it's going to be displayed in the running processes.
ide-flatpak-wrapper call ide by fork, so it's not used that name

using "-wrapper" is usefull to distinguish the wrapper from the original ide